config/config.go

```diff
@@ -40,6 +40,7 @@ type Config struct {
 	TrustSystemCACerts               bool   // If equal to true, the HTTP client factory will set up clients that trust CA certs provided by a Linux distribution (see https://golang.org/pkg/crypto/x509/#SystemCertPool and https://golang.org/src/crypto/x509/root_linux.go)
 	CACertsPath                      string // Path to a file containing PEM-encoded x509 certs HTTP clients in Anax will trust (additive to the configuration option "TrustSystemCACerts")
 	ExchangeURL                      string
+	AgbotURL                         string
```

Comment:
Currently all /etc/default/horizon file has HZN_AGBOT_URL. Please fill the value of AgbotURL from the env HZN_AGBOT_URL in function `enrichFromEnvvars()`. See how CSS url is filled.